ArjunaJTS/interop/test-ejbs/pom.xml

```diff
@@ -147,29 +129,5 @@
          </plugin>
       </plugins>
    </build>
-
-   <profiles>
-      <profile>
-         <!-- When built in OpenShift the 'openshift' profile will be used when invoking mvn. -->
-         <!-- Use this profile for any OpenShift specific customization your app will need. -->
-         <!-- By default that is to put the resulting archive into the 'deployments' folder. -->
-         <!-- http://maven.apache.org/guides/mini/guide-building-for-different-environments.html -->
-         <id>openshift</id>
-         <build>
-            <plugins>
-               <plugin>
-                  <artifactId>maven-war-plugin</artifactId>
-                  <version>2.1.1</version>
-                  <configuration>
-                     <outputDirectory>deployments</outputDirectory>
-                     <warName>ROOT</warName>
-                     <!-- Java EE 6 doesn't require web.xml, Maven needs to catch up! -->
-                     <failOnMissingWebXml>false</failOnMissingWebXml>
-                  </configuration>
-               </plugin>
-            </plugins>
-         </build>
-      </profile>
-   </profiles>
```

Comment:
Why have you removed this profile?

Reply:
Thanks @jmfinelli! I removed it because this profile only has the maven-war-plugin (present in root pom). But I can see it has specific configuration so I am going to restore the profile.